Source/WebKit/Shared/mac/PDFContextMenu.h

```diff
@@ -28,13 +28,17 @@
 #if ENABLE(PDF_PLUGIN) && PLATFORM(MAC)
 
 namespace WebKit {
+enum class ContextMenuItemEnablement : bool { Disabled, Enabled };
+
+enum class ContextMenuItemIsSeparator : bool { No, Yes };
     
+enum class ContextMenuItemHasAction : bool { No, Yes };
 struct PDFContextMenuItem {
     String title;
-    bool enabled;
-    bool separator;
+    ContextMenuItemEnablement enabled;
+    ContextMenuItemIsSeparator separator;
     int state;
-    bool hasAction;
+    ContextMenuItemHasAction hasAction;
```

Comment:
Please group all the enums together for better padding.

Reply:
Done. Also added replaced a `static_cast` with `enumToUnderlyingType()`